packages/ag-charts-enterprise/src/axes/radius-category/radiusCategoryAxis.ts

```diff
@@ -0,0 +1,200 @@
+import type { AgAxisCaptionFormatterParams } from 'ag-charts-community';
+import { _ModuleSupport, _Scale, _Scene, _Util } from 'ag-charts-community';
+import { RadiusCrossLine } from './radiusCrossLine';
+
+const {
+    AND,
+    assignJsonApplyConstructedArray,
+    ChartAxisDirection,
+    Default,
+    GREATER_THAN,
+    Layers,
+    LESS_THAN,
+    NUMBER,
+    NUMBER_OR_NAN,
+    OPT_BOOLEAN,
+    ProxyPropertyOnWrite,
+    Validate,
+} = _ModuleSupport;
+const { BandScale } = _Scale;
+const { Arc, Caption, Group, Selection } = _Scene;
+const { toRadians } = _Util;
+
+class RadiusCategoryAxisTick extends _ModuleSupport.AxisTick<_Scale.LinearScale, number> {
+    @Validate(AND(NUMBER_OR_NAN(1), GREATER_THAN('minSpacing')))
+    @Default(NaN)
+    maxSpacing: number = NaN;
+}
+
+class RadiusCategoryAxisLabel extends _ModuleSupport.AxisLabel {
+    @Validate(OPT_BOOLEAN)
+    autoRotate: boolean | undefined = undefined;
+
+    @Validate(NUMBER(-360, 360))
+    autoRotateAngle: number = 335;
+}
+
+type TickDatum = {
+    tickLabel: string;
+    tick: any;
+    tickId: string;
+    translationY: number;
+};
+
+export class RadiusCategoryAxis extends _ModuleSupport.PolarAxis {
```

Comment:
There seems to be a bunch of commonality with `radiusNumberAxis.ts` - can we also factor out some common code?